autobahn/wamp_error.hpp

```diff
@@ -49,8 +49,8 @@ class wamp_error : public std::runtime_error {
         , m_details(other.m_details)
         , m_args(other.m_args)
         , m_kw_args(other.m_kw_args)
-        , m_zone(other.m_zone)
     {
+        m_zone.swap(other.m_zone);
```

Comment:
I would give that a qualified yes. Typically, I wouldn't normally do that, but the only place that ever gets called is from a temporary object, like an rvalue reference, in which case it is safe. It is probably worth looking for a better way. I like this better than the shared pointer tho.